src/javascripts/components/copy.mjs

```diff
@@ -29,11 +34,11 @@ class Copy {
     this.$status.className = 'govuk-visually-hidden'
     this.$status.setAttribute('aria-live', 'assertive')
 
-    this.$module.insertAdjacentElement('beforebegin', this.$status)
-    this.$module.insertAdjacentElement('beforebegin', this.$button)
+    this.$module.prepend(this.$status)
+    this.$module.prepend(this.$button)
```

Comment:
**note** `$module` is no longer the `<pre>` element containing the code, but [the `<div>` wrapping it](https://github.com/alphagov/govuk-design-system/blob/b56ee4348f260434e239dd53b79631c68147a604/views/partials/_example.njk#L65), which simplifies the insertion of the button and hidden status for assistive technology.